ansible/my-ansible-playbook.yml

```diff
@@ -3,11 +3,8 @@
   remote_user: ec2-user
 
   vars_prompt:
-  - name: trello_api_key
-    prompt: What is your TRELLO API KEY?
-
-  - name: trello_api_token
-    prompt: What is your TRELLO API TOKEN?
+  - name: mongodb_primary_connection_string
+    prompt: What is your PRIMARY CONNECTION STRING for MongoDB?
```

Comment:
Nice - don't forget it's not _required_ to maintain these Ansible files (since we won't be reusing them) but equally it's a good habit to be in!